doc/en/user/source/community/wps-download/rawDownload.rst

```diff
@@ -34,6 +34,7 @@ The parameters to set are
  * ``targetSizeX`` : size X in pixels of the output (optional, applies for raster input only)
  * ``targetSizeY`` : size Y in pixels of the output (optional, applies for raster input only)
  * ``selectedBands`` : a set of the band indices of the original raster that will be used for producing the final result (optional, applies for raster input only)
+ * ``writeParameters`` : a set of writing parameters (optional, applies for raster input only)
```

Comment:
Since this is a complex input, I'd refer people to the example at the bottom of document to show how they are encoded

Reply:
Done